extras/iris_pipeline_project/README.md

```diff
@@ -0,0 +1,126 @@
+# OVMS Iris Pipeline Example
+
+This repository demonstrates how to use OpenVINO Model Server (OVMS) with a custom Mediapipe pipeline for the Iris dataset, including both model training and inference through a Python client.
+
+---
+
+## Step 1: Clone the Repository
+
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server/extras/iris_pipeline_project
+```
+---
+
+
+## Step 2: Build and Run OVMS Docker Image
+
+### 2.1. Build the Docker Image
+
+```bash
+docker build --no-cache -t iris_logisticreg_ovms .
+```
+
+### 2.2. Run the OVMS Container
+
+```bash
+docker run --rm -it -v "$PWD:/workspace"   -p 9000:9000 -p 8000:8000  iris_logisticreg_ovms --config_path /workspace/model_config.json   --port 9000 --rest_port 8000
+```
+- **Note:** Adjust `$(pwd)` if you are running from a different working directory.
+
+---
+
+##  Step 3: Project Structure
+
+```
+client/
+  ├── client_inference.py
+  └── client_train.py
+pipeline/
+  ├── __pycache__/
+  ├── graph.pbtxt
+  └── ovmsmodel.py
+Dockerfile
+model_config.json
+```
+
+---
+
+## Step 4: Run Training and Inference
+
+### 4.1. Training
+
+```bash
+python client/client_train.py train <path_to_training_dataset>
+```
+
+### 4.2. Inference
+
+```bash
+python client/client_inference.py infer <path_to_test_dataset>
+```
+
+---
+
+## Instructions for preparing the data
+Run the command to download the Iris dataset, which is taken to be the hello-world dataset of classification datasets.
+
+```bash
+curl -o iris.csv https://archive.ics.uci.edu/ml/machine-learning-databases/iris/iris.data
+```
+Run the following file to prepare the data and split it into data for training and for inferencing.
+
+```bash
+python data_preprocess.py <path to output_dir>
+```
```

Comment:
Please make it a separate step before step 4, so user can follow the guide step-by-step